plugins/aws/utils.go

```diff
@@ -20,3 +24,12 @@ func getConfigSectionByProfile(configFile *ini.File, profileName string) *ini.Se
 
 	return nil
 }
+
+func ExecuteSilently(f func() (*vault.ConfigFile, error)) func() (*vault.ConfigFile, error) {
+	return func() (*vault.ConfigFile, error) {
+		log.SetOutput(io.Discard)
+		vaultConfig, err := f()
+		defer log.SetOutput(os.Stderr)
+		return vaultConfig, err
+	}
+}
```

Comment:
Let's please use the example I wrote in  https://github.com/1Password/shell-plugins/pull/297#discussion_r1232187265 which uses go generics, and is not constrained by the `(*vault.ConfigFile, error)` return types.

The reason I'm insisting for a more generic function is because we need to apply it in multiple other places as per my next other comment.